docs/excel/excel-add-ins-worksheets.md

```diff
@@ -382,18 +382,20 @@ await Excel.run(async (context) => {
 
 ## Find all cells with matching text
 
-The `Worksheet` object has a `find` method to search for a specified string within the worksheet. It returns a `RangeAreas` object, which is a collection of `Range` objects that can be edited all at once. The following code sample finds all cells with values equal to the string **Complete** and colors them green. Note that `findAll` will throw an `ItemNotFound` error if the specified string doesn't exist in the worksheet. If you expect that the specified string may not exist in the worksheet, use the [findAllOrNullObject](../develop/application-specific-api-model.md#ornullobject-methods-and-properties) method instead, so your code gracefully handles that scenario.
+The `Worksheet` object has a [`findAll` method](/javascript/api/excel/excel.worksheet#excel-excel-worksheet-findall-member(1)) to search for a specified string within the worksheet. It returns a `RangeAreas` object, which is a collection of `Range` objects that can be edited all at once.
```

Comment:
Elsewhere in this article the descriptor ("method", "object") isn't included in the link text. 
```suggestion
The `Worksheet` object has a [`findAll`](/javascript/api/excel/excel.worksheet#excel-excel-worksheet-findall-member(1)) method to search for a specified string within the worksheet. It returns a `RangeAreas` object, which is a collection of `Range` objects that can be edited all at once.
```